torch/testing/_internal/common_methods_invocations.py

```diff
@@ -454,6 +459,8 @@ def sample_inputs(self, device, dtype, requires_grad=False):
                        SkipInfo('TestUnaryUfuncs', 'test_reference_numerics',
                                 device_type='cuda', dtypes=[torch.cfloat, torch.cdouble],
                                 active_if=IS_WINDOWS),
+                       SkipInfo('TestSparseUnaryUfuncs', 'test_sparse_consistency',
```

Comment:
Why are these skips needed?

Reply:
Currently, there are few inconsistencies while using complex and bfloat16 dtype like:
- coalesce() fails on bfloat16
- to_sparse() fails on complex

Probably, should be filed as issues and referenced here.

Reply:
Thanks for clarifying, @vfdev-5. In that case I'd like to propose that the test skip these dtypes instead of the OpInfo skipping them, since it's the test that's failing, not the operator.

Reply:
@mruberry - As mentioned by @vfdev-5 ; the skips are added for precisely that reason; will modify the framework to skip this dtypes and remove from the opinfo

Reply:
You don't need to modify the framework. Just check if one of these dtypes is the dtype passed to the test and skip the test if that's the case.

Reply:
> You don't need to modify the framework. Just check if one of these dtypes is the dtype passed to the test and skip the test if that's the case.

Have done the same